server/service/src/sync/test/test_data/invoice.rs

```diff
@@ -98,119 +98,125 @@ const TRANSACT_1: (&str, &str) = (
       "om_expected_delivery_date": ""
   }"#,
 );
+
+fn transact_1_pull_row() -> InvoiceRow {
```

Comment:
wanted to re-use this for the test, so I don't have to re-define a lot of fields, probably would have been quicker if i copied it